modules/gapi/src/executor/gstreamingexecutor.cpp

```diff
@@ -1443,6 +1491,13 @@ void cv::gimpl::GStreamingExecutor::setSource(GRunArgs &&ins)
             out_queues.push_back(reader_queues(*m_island_graph, out_eh));
         }
 
+        // Create just empty island meta information
+        std::string island_meta_info { };
+#if defined(OPENCV_WITH_ITT) && !defined(GAPI_STANDALONE)
```

Comment:
btw whats wrong with STANDALONE here?

Reply:
Fixed!
Thanks a lot!
I have realized that `OPENCV_WITH_ITT` will be defined only if ITT dependency will be successfully resolved, what can be done only if we building with OpenCV. So, there will be no such define in G-API standalone mode and check for `!GAPI_STANDALONE` is extra.
I have added a comment about this in `itt.hpp` file and `TODO` to support ITT traces in GAPI standalone mode in `CMakeLists.txt`.